ledger/caching/src/main/scala/com/daml/caching/Cache.scala

```diff
@@ -37,6 +37,8 @@ abstract class Cache[Key, Value] {
       mapBeforeWriting: NewValue => Option[Value],
   ): Cache[Key, NewValue] =
     new MappedCache(mapAfterReading, mapBeforeWriting)(this)
+
+  def invalidate(key: Key): Unit
```

Comment:
would be nice to add a unit test for the `invalidate`

Reply:
Added

Reply:
Dropped as not needed